README.md

```diff
@@ -1,70 +1,57 @@
-# Getting Started with Create React App
+# Weather Forecast
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+ This web application fetches weather forecast data from https://open-meteo.com/en/docs API for cities displayed on the homepage, the user can also choose specific country and then the city to view cooresponding temperature data.
 
-## Available Scripts
+ ## Live link
+ [Link](https://elastic-turing-eb476d.netlify.app/)
 
```

Comment:
- [ ] Please record a video which will you tell about the project.

![Capture d’écran 2021-11-27 130413](https://user-images.githubusercontent.com/61562626/143680621-e9ca4fd5-ee15-4cbb-a41c-fe30a6b37d67.png)